packages/botbuilder-adapter-slack/src/slack_adapter.ts

```diff
@@ -242,9 +242,14 @@ export class SlackAdapter extends BotAdapter {
      * @returns A url pointing to the first step in Slack's oauth flow.
      */
     public getInstallLink(): string {
+        let redirect = ''
         if (this.options.clientId && this.options.scopes) {
-            let redirect = 'https://slack.com/oauth/authorize?client_id=' + this.options.clientId + '&scope=' + this.options.scopes.join(',');
-
+            if (this.options.oauthVersion == 'v2'|'V2'){
```

Comment:
```suggestion
            if (this.options.oauthVersion == 'v2'||'V2'){
```

Reply:
That is not a valid  conditional.

that is basically saying
`if ((this.options.authversion==v2) || ('V2'))`

it would need to be 2 separate conditions.

but i have done this in a different way so don't worry about it./